html/user/get_project_config.php

```diff
@@ -18,6 +18,8 @@
 
 require_once("../inc/util.inc");
 require_once("../inc/xml.inc");
+if(file_exists('../../release.inc'))
+    include '../../release.inc';
```

Comment:
Do you test for file existence simply to reduce the warning message that may occur using the include?

Also - I think the practice used in BOINC is to go ahead and use include_once even though it isn't strictly necessary in this case.